lib/utils/background_push.dart

```diff
@@ -50,16 +50,15 @@ class BackgroundPush {
   final FlutterLocalNotificationsPlugin _flutterLocalNotificationsPlugin =
       FlutterLocalNotificationsPlugin();
   Client client;
-  BuildContext? context;
-  GlobalKey<NavigatorState>? router;
+  GlobalKey<NavigatorState>? globalKey;
```

Comment:
```suggestion
  GlobalKey<NavigatorState>? navigatorKey;
```

Reply:
IMO: 
- I think `navigatorKey ` is wrongly, just `NavigatorState `

Reply:
GlobalKey of NavigatorState = navigatorKey

Reply:
or just keep `router` as the origin

Reply:
I'll change `navigatorKey ` to `globalNavigatorKey`